### ISSUE TITLE
no post_url, skipping

### ISSUE THREAD

**Issue by saqtam66** [shaikhsajid1111/facebook_page_scraper#115]
Hello
When running the scarper: i got the following error "no post_url, skipping" repeatdly,
no post scraped
I am using "firefox" browser.
Is there a solution?

**Comment by moda20** [comment]
@saqtam66 It seems this repo doesn't find post urls anymore, facebook changed their website

**Comment by shaikhsajid1111** [comment]
@moda20 that's right. Facebook has had drastic changes in its UI. I'm trying to figure out if there's currently a way

**Comment by moda20** [comment]
@shaikhsajid1111 I created a fork that can parse the url and postTime and other identifiers : 
https://github.com/moda20/facebook_page_scraper

**Comment by shaikhsajid1111** [comment]
@moda20 I see crawler hovering over dates but still in result it is not available. Also, post Ids

**Comment by shaikhsajid1111** [comment]
Well, I have found a way. Probably be pushing totally a new way under a major release

**Comment by Reeyomma** [comment]
When will the next release be? Having this same issue.
new layout loaded
all_posts length: 4
no post_url, skipping
no post_url, skipping
no post_url, skipping
no post_url, skipping
all_posts length: 4
all_posts length: 7
no post_url, skipping

**Comment by moda20** [comment]
@Reeyomma try my fork, https://github.com/moda20/facebook_page_scraper, it's drop in replacement. 
it works for me for now,

**Comment by shaikhsajid1111** [comment]
I found a way through Facebook's GraphQL but haven't had enough time to spend on writing the parser for it

**Comment by shaikhsajid1111** [comment]
Here's the response (Attached File). I'm trying to understand if it can resolve the issue.
[graphql_response.json](https://github.com/user-attachments/files/16225761/graphql_response.json)

**Comment by Irene933** [comment]
Printing out status link I noticed that links included in post content get affected to post_url/status_link/etc instead of the actual post url.
